src/components/injector.tsx

```diff
@@ -24,6 +24,16 @@ import {bindActionCreators} from 'redux';
 import {console as Console} from 'global/window';
 import KeplerGlContext from 'components/context';
 
+export type FactoryElement = (...args) => React.Component;
```

Comment:
shoud this be `React.ComponentType` instead of `React.Component`? so It also includes `React.FC`?